src/crosschain/swapExactIn/thorChainSwap/index.ts

```
@@ -1,2 +1,3 @@
 export { thorChainSwap } from './thorChainSwap'
-export { THOR_TOKENS_IN } from './utils'
+export { THORCHAIN_TOKENS_EVM_TRANSIT as THORCHAIN_TOKENS_IN, THORCHAIN_DESTINATIONS, THORCHAIN_L1_DEST_CHAIN_IDS as THORCHAIN_L1_CHAIN_IDS } from './constants'
```

ESLint

[error] Replace `·THORCHAIN_TOKENS_EVM_TRANSIT·as·THORCHAIN_TOKENS_IN,·THORCHAIN_DESTINATIONS,·THORCHAIN_L1_DEST_CHAIN_IDS·as·THORCHAIN_L1_CHAIN_IDS·` with `⏎····THORCHAIN_TOKENS_EVM_TRANSIT·as·THORCHAIN_TOKENS_IN,⏎····THORCHAIN_DESTINATIONS,⏎····THORCHAIN_L1_DEST_CHAIN_IDS·as·THORCHAIN_L1_CHAIN_IDS,⏎`

(prettier/prettier)